Source/Particles/WarpXParticleContainer.cpp

```diff
@@ -233,14 +233,14 @@ WarpXParticleContainer::AddNParticles (int /*lev*/,
  *                       time of the deposition.
  */
 void
-WarpXParticleContainer::DepositCurrent(WarpXParIter& pti,
-                                       RealVector& wp, RealVector& uxp,
-                                       RealVector& uyp, RealVector& uzp,
-                                       const int * const ion_lev,
-                                       MultiFab* jx, MultiFab* jy, MultiFab* jz,
-                                       const long offset, const long np_to_depose,
-                                       int thread_num, int lev, int depos_lev,
-                                       Real dt, Real relative_time)
+WarpXParticleContainer::DepositCurrent (WarpXParIter& pti,
+                                        RealVector const & wp, const RealVector const & uxp,
+                                        RealVector const & uyp, const RealVector const & uzp,
```

Comment:
I think the compiler complains about too many `const` here:
```suggestion
                                        RealVector const & wp, RealVector const & uxp,
                                        RealVector const & uyp, RealVector const & uzp,
```

Reply:
Ups, yes that's a mistake :)